core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.ts

```diff
@@ -96,6 +96,7 @@ export class TopNavigationBarComponent implements OnInit, OnDestroy {
   donateMenuOffset: number = 0;
   ACTION_OPEN!: string;
   ACTION_CLOSE!: string;
+  private backButtonHiddenPaths: string[] = ['/blog/'];
```

Comment:
```suggestion
  PAGES_WITHOUT_BACK_STATE: string[] = ['/blog/'];
```

Also, move this to the top (below `IMPACT_REPORT_LINK`.)

Reply:
Done.